template/Data-Science/Prediction/Diabetes-Prediction/model_trainning_and_saving.py

```
@@ -0,0 +1,106 @@
+    print(f'Accuracy score : {model_data[0]}')
+    print(f'Confusion Matrix : \n{model_data[1]}')
+    print(f'Classification Report : \n{model_data[2]}')
+
+
+
+model_objects = {
+    'preprocessor': preprocessor,
+    'models': models
+}
+
+file = open('model_objects.pkl','wb')
```

Ruff

Use a context manager for opening files

(SIM115)